benchmarks/dijkstra/functional_c#/Program.cs

```diff
@@ -106,32 +113,38 @@ private static ImmutableList<string> dijkstraHelper(ImmutableDictionary<string,
             }
         }
 
-        private static (ImmutableSortedSet<(string, int, string)>, ImmutableDictionary<string, (string, int, string)>) updateCosts(ImmutableDictionary<string, ImmutableList<(string, string, int)>> graph, (string, int, string) currentVertex, ImmutableSortedSet<(string, int, string)> vertexQueue, ImmutableDictionary<string, (string, int, string)> visited, ImmutableDictionary<string, (string, int, string)> vertexCosts)
+        private static (ImmutableSortedSet<(string, int, string)>, ImmutableDictionary<string, (string, int, string)>) updateCosts(
+            ImmutableDictionary<string, ImmutableArray<(string source, string dest, int cost)>> graph, 
+            (string name, int costToReach, string previous) currentVertex, 
+            ImmutableSortedSet<(string, int, string)> vertexQueue, 
+            ImmutableDictionary<string, (string, int, string)> visited, 
+            ImmutableDictionary<string, (string name, int costToReach, string previous)> vertexCosts
+            )

```

Comment:
Tror den her tager førstepladsen for længste funktionssignatur indtil videre. xD

Reply:
Jep XD